src/types.rs

```diff
@@ -147,6 +149,12 @@ pub struct WgServerConfig {
 fn default_gateway_ip_v4() -> net::Ipv4Addr {
     net::Ipv4Addr::new(10, 64, 0, 1)
 }
+fn default_relay_addr_v4() -> net::SocketAddrV4 {
+    net::SocketAddrV4::new(net::Ipv4Addr::LOCALHOST, 1000)
+}
+fn default_relay_addr_v6() -> net::SocketAddrV6 {
+    net::SocketAddrV6::new(net::Ipv6Addr::LOCALHOST,1000, 0, 0)
+}
```

Comment:
I don't think we should set a fake default. We should keep setting it properly on the server until it is removed.  Once it is removed I would rather error out then get bogus data.

Maybe what we need is a `server` feature that enables these fields. Then the client won't have the field at all but the server is still forced to set them.

Reply:
I didn't think of using a feature. That's much better. The deprecation warnings on the server would have been annoying too.

Without a feature I didn't see an alternative to fake defaults. We wouldn't stop setting it on the server until it is removed either way.

Reply:
Done. The only downside I can see so far is that we have to remember to run tests with all features enabled `cargo test --all-features`.

Reply:
As long as that is being done in CI we can't really forget.